### ISSUE TITLE
is this repo cloesd?

### ISSUE THREAD

**Issue by Alan2772** [ishann/aeroscapes#4]
I think this is a good repo for image segmentation, takes a great distance 5-50m.
would you like to relaunch this repo again? posting the dataset and some codes.

**Comment by ishann** [comment]
Hey, apologies for the delay in replying.
The data is now hosted on Google Drive. Please see the updated README.

As the paper indicates, the training strategy is quite simple. We freeze the task-specific layers in pre-trained models and finetune them on the AeroScapes dataset. A linear regressor is trained to combine the softmax predictions from each network. This strategy should allow you to train your own network. Unfortunately, it is not possible for us to release the code/ models.
